pkg/kubelet/cm/dra/manager.go

```diff
@@ -198,34 +184,40 @@ func (m *ManagerImpl) PrepareResources(pod *v1.Pod, container *v1.Container) (*C
 	return &ContainerInfo{Annotations: annotations}, nil
 }
 
-func (m *ManagerImpl) UnprepareResources(pod *v1.Pod) error {
-	m.Lock()
-	defer m.Unlock()
+// PrepareResources calls plugin NodePrepareResource from the registered DRA resource plugins.
+func (m *ManagerImpl) PrepareResources(pod *v1.Pod, container *v1.Container) (*ContainerInfo, error) {
+	if err := m.prepareContainerResources(pod, container); err != nil {
+		return nil, err
+	}
+
+	return m.getContainerInfo(pod, container)
+}
 
+func (m *ManagerImpl) UnprepareResources(pod *v1.Pod) error {
 	// Call NodeUnprepareResource RPC for every resource claim referenced by the pod
 	for _, podResourceClaim := range pod.Spec.ResourceClaims {
 		claimName := resourceclaim.Name(pod, &podResourceClaim)
 
-		resource := m.resources.get(claimName, pod.Namespace)
+		resource := m.cache.get(claimName, pod.Namespace)
```

Comment:
variable name should be `claimInfo`now

Reply:
We renamed resource -> claimInfo in the resources.go:claimInfoCache
Are you suggesting to rename cache to claimInfo in the manager.go:claimInfoCache as well?

Reply:
cache looks ok to me in the ManagerImpl. I wouldn't restrict it to hold only claimInfo. We can put other information in it in future.

Reply:
ah, sorry. will do, thanks.

Reply:
The cache can hold whatever it wants, but the return type of `m.cache.get` is currently a `claimInfo` object, so I'd like that to be better represented in the variable name that holds that.

If it helps for you to rename the function to `getClaimInfo()` as part of this, then I'm fine with that too.

The variable name `resource` implies something else to me, and is confusing to see when reading the code as an outsider, given that we are dealing with `resourceClaims` (and not the resources themselves) throughout this component.

Reply:
btw, would it make sense to rename resources.go -> cache.go ?

Reply:
Sure or claimInfo.go

Reply:
done.